e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/generate-keying-sheet.spec.js

```diff
@@ -178,12 +178,15 @@ context('PDC_RECONCILE users can generate keying data', () => {
      * current fixed fee is given by the product of the below
      * values:
      * - utilisation: 180000
+     * - cover percentage: 40
+     * - ukef utilisation: 180000 * 0.4 = 72000
      * - bank margin: 0.9 (this is a fixed, constant value)
      * - interest percentage: 0.05
      * - number of days left in cover period divided by day count basis: 365 / 365 = 1
-     * This yields a current utilisation value of 8100
+     * - (72000 * 0.9 * 0.05 * 1) = 3240
```

Comment:
Not a blocker. Seems we have 2/3 places to maintain such documentation instead of just calling a fucntion in tests...